.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -0,0 +1,27 @@
+---
+name: Feature Request
+about: Suggest an idea for this project
+title: "[FEATURE REQUEST]"
+labels: enhancement
+assignees: ''
+
+---
+
+## Is your feature request related to a problem? If so, please give a short summary of the problem and how the feature would resolve it.
+<!--- A clear and concise description of what the problem is. Ex. I'm always frustrated when [...] --->
+
+## Describe the preferred solution.
+<!--- A clear and concise description of what you want to happen. --->
+
+## Describe alternatives you've considered
```

Comment:
Should this go under optional?

Reply:
That ways everything is optional.. but knowing alternatives kind of tells us where the idea is coming from and is more probable to mention... so I'll keep it as it is.